### PR TITLE
Tj/merchant handlers

### PR DESCRIPTION
**Pull request by littlejohntj**

### CODE REVIEW

**Review by harshasomisetty — APPROVED** [review]

File: apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/merchant-data.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        if (merchant == null) {
+            return requestErrorResponse(new Error('Merchant not found'));
+        }
+
+        const generalResponse = await createGeneralResponse(merchantAuthToken, prisma);
+        const onboardingResponse = createOnboardingResponse(merchant);
+
+        // TODO: Create a type for this
+        const responesBodyData = {
```
Comment (harshasomisetty) [inline apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/merchant-data.ts:39]: responseBodyData

File: apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/payment-data.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+        const pagination: Pagination = {
+            page: paymentDataRequestParameters.page,
+            pageSize: paymentDataRequestParameters.pageSize,
+        };
+
+        const generalResponse = await createGeneralResponse(merchantAuthToken, prisma);
+        const paymentResponse = await createPaymentResponse(merchantAuthToken, pagination, prisma);
+
+        const responesBodyData = {
```
Comment (harshasomisetty) [inline apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/payment-data.ts:57]: responseBodyData

File: apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/refund-data.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+        const pagination: Pagination = {
+            page: refundDataRequestParameters.page,
+            pageSize: refundDataRequestParameters.pageSize,
+        };
+
+        const refundResponse = await createRefundResponse(merchantAuthToken, 'pending', pagination, prisma);
+        const generalResponse = await createGeneralResponse(merchantAuthToken, prisma);
+
+        const responesBodyData = {
```
Comment (harshasomisetty) [inline apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/refund-data.ts:57]: responseBodyData

Comment (littlejohntj) [reply apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/refund-data.ts]: updated

File: apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/payment-data.ts (hunk 2 above)
Comment (littlejohntj) [reply apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/payment-data.ts]: updated

File: apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/merchant-data.ts (hunk 1 above)
Comment (littlejohntj) [reply apps/backend-serverless/src/handlers/clients/merchant-ui/read-data/merchant-data.ts]: updated

**Pull request merged**